### PR TITLE
MCKIN-11588 Set default course image url to empty

### PR DESCRIPTION
**Pull request by moeez96**
Jira ticket: https://edx-wiki.atlassian.net/browse/MCKIN-11588
Set default course image url to empty.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @moeez96! I've created [OSPR-4307](https://openedx.atlassian.net/browse/OSPR-4307) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://open.edx.org/wp-content/uploads/2019/01/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

File: openedx/core/lib/courses.py [hunk 1]
```diff
@@ -27,8 +27,8 @@ def course_image_url(course, image_key='course_image'):
         else:
             url += '/images/' + image_key + '.jpg'
     elif not getattr(course, image_key):
-        # if image_key is empty, use the default image url from settings
-        url = settings.STATIC_URL + settings.DEFAULT_COURSE_ABOUT_IMAGE_URL
+        # if image_key is empty, set empty url by default
+        url = ""
```
Comment (naeem91) [inline openedx/core/lib/courses.py]: Is broken image an issue? In that case we can ensure a correct default image exists instead of making this change?

Comment (nasirhjafri) [reply openedx/core/lib/courses.py]: I agree with @naeem91. Also, It was decided to use this image as default image as part of UI Refresh https://github.com/mckinseyacademy/mcka_apros/blob/development/static_v2/image/default-course-image.png 

Let's add this image in static files instead.

Comment (moeez96) [reply openedx/core/lib/courses.py]: Thanks for the suggestions. An authoring based solution was devised to the reporter using the default image provided above.

**Comment by moeez96** [comment]
An authoring based solution was devised to the reporter using the default image provided above.
Closing this PR.

**Pull request closed without merging**